pywbemtools/pywbemcli/pywbemcli.py

```diff
@@ -124,6 +124,58 @@ def execute_startup_script(file_path, verbose):
                 err=True)
             raise click.Abort()
 
+def _resolve_mock_server(mock_server):
+    """
+    Validate and resolve the mock_server parameter.
+    Validates that all files defined in the server exist and that the
+    types are valid.
+
+    TODO: Confirm that it is necessary to join the basename to cwd in the
+    case only basename is provided for all OSs.
+
+    Parameters:
+
+      mock_server (list of _resolve_mock_server):
+        Strings defining all of the files that compromise the mock_server
+        definiton.
+
+    Returns: list of resolved mock server definition components derived
+       from the input parameter mock_server.
+
+    Raises:
+      click.ClickException for any errors.
+    """
+    resolved_mock_server = []
+    assert isinstance(mock_server, tuple)
+    # resolve relative and absolute paths
+    mock_server_path = []
+    for fn in mock_server:
+        if fn == os.path.basename(fn):
+            mock_server_path.append(os.path.join(os.getcwd(), fn))
+        else:
+            mock_server_path.append(fn)
```

Comment:
So if the path is a relative path it stays relative, but if it is just a base file name we prepend the cwd?

I think that prepending is not needed at all and just complicates things. The specified file paths are supposed to be reachable from the current directory, and there is no need to prepend anything.

Reply:
Redid this.  Actually changed it to just run all filenames through the normpath.

Reply:
Another new GitHub feature: Outdated comments no longer seem to show up in the file view even when they are not resolved.

On your use of `os.path.normpath()`: Its description says "This string manipulation may change the meaning of a path that contains symbolic links.". So while I like the idea, it is potentially problematic.

Why do we do anything at all? The user ought to specify a valid path name. If the file path references a file then it works and if not then it fails. That works on all OS types with all variations on how file paths can be specified. The only downside of non-normalized file paths is that the caching mechanism will not consider two file paths as equal that reference the same file in different ways. But that is only a very small problem, IMO.